### ISSUE TITLE
Add GoodBuyDPI in to adguard 

### ISSUE THREAD

**Issue by Eksol** [AdguardTeam/AdGuardVPNForAndroid#483]
### Issue Details

There are 2 options to solve the slow speed of YouTube.
1. Use a VPN.
2. Use goodbpdpi.
The second option is faster.
The problem is that you can't create 2 VPN connections on Android.
Both solutions use the creation of a local VPN to intercept traffic.

Because of this, we have a choice between.
1. Fast YouTube but ads on sites
2. Non-working YouTube but no ads on sites.

### Proposed solution

Add a small addition to the Adguard application that would allow you to selectively overclock YouTube without losing the functionality of Adguard.

**Comment by sfionov** [comment]
@Eksol AdGuard for Android already have Anti-DPI functionality with the most common settings implemented (split packets, split delay, increase first packet size).

**Comment by maxikuzmin** [comment]
@Eksol did you try this with the Anti-DPI setting?

**Comment by Eksol** [comment]
> did you try this with the Anti-DPI setting?
Yes, I tried to enable this feature on the Windows version. But it doesn't work.

**Comment by maxikuzmin** [comment]
@Eksol well, we'll look into the problem, see what we can do
